src/mds/CDir.cc

```diff
@@ -2421,6 +2421,13 @@ void CDir::_omap_commit_ops(int r, int op_prio, int64_t metapool, version_t vers
   auto commit_one = [&](bool header=false) {
     ObjectOperation op;
 
+    /*
+     * just skip submiting the invalidate request, which could cause
```

Comment:
@lxbsz According to the tracker, the issue happend after the upgrade. I didn't quite understand why the invalid osd_op is sent for the large metadata inode only after upgrade ? Could you please explain ?

Reply:
@kotreshhr Good question.

Before upgrading it also could send for the large metadata requests and mostly it will be okay and won't trigger this bug. But it's very probably that the first item in the `to_set` list is a large one. It's buggy when the `to_set` first item really has a large size it will try to send the osd request to Rados before moving this item to `_set` list.

The root dir seems have only one dentry, which is the `/volumes`, so after upgraded when starting the MDS it will trigger it.